framework/src/microservice.js

```diff
@@ -125,22 +125,27 @@ const Microservice = (config = {}) => {
 			return;
 		}
 
-		if (!job.schedule && !job.interval) {
+		if ((job.controller && !job.schedule && !job.interval)) {
```

Comment:
Kindly consider adding another check that enforces that either `init` or `controller` is required to be present. This is important since we've updated all the fields within the `jobSchema` to be `optional`.

Reply:
Added one extra check to ensure this.